### ISSUE TITLE
2025-03-10, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#106]
# Today's Arxiv Papers

(1) [Understanding the Limits of Lifelong Knowledge Editing in LLMs](http://arxiv.org/abs/2503.05683v1), 2025-03-07

(2) [AIM-Fair: Advancing Algorithmic Fairness via Selectively Fine-Tuning Biased Models with Contextual Synthetic Data](http://arxiv.org/abs/2503.05665v1), 2025-03-07

(3) [A Survey of Large Language Model Empowered Agents for Recommendation and Search: Towards Next-Generation Information Retrieval](http://arxiv.org/abs/2503.05659v1), 2025-03-07

(4) [Symbolic Mixture-of-Experts: Adaptive Skill-based Routing for Heterogeneous Reasoning](http://arxiv.org/abs/2503.05641v1), 2025-03-07

(5) [Learning LLM Preference over Intra-Dialogue Pairs: A Framework for Utterance-level Understandings](http://arxiv.org/abs/2503.05620v1), 2025-03-07

(6) [A Survey on Sparse Autoencoders: Interpreting the Internal Mechanisms of Large Language Models](http://arxiv.org/abs/2503.05613v1), 2025-03-07

(7) [R1-Searcher: Incentivizing the Search Capability in LLMs via Reinforcement Learning](http://arxiv.org/abs/2503.05592v1), 2025-03-07

(8) [Quantifying the Robustness of Retrieval-Augmented Language Models Against Spurious Features in Grounding Data](http://arxiv.org/abs/2503.05587v1), 2025-03-07

(9) [Evaluating open-source Large Language Models for automated fact-checking](http://arxiv.org/abs/2503.05565v1), 2025-03-07

(10) [Revitalizing Saturated Benchmarks: A Weighted Metric Approach for Differentiating Large Language Model Performance](http://arxiv.org/abs/2503.05551v1), 2025-03-07

(11) [Pi-GPS: Enhancing Geometry Problem Solving by Unleashing the Power of Diagrammatic Information](http://arxiv.org/abs/2503.05543v1), 2025-03-07

(12) [Leveraging Approximate Caching for Faster Retrieval-Augmented Generation](http://dx.doi.org/10.1145/3721146.3721941), 2025-03-07

(13) [PoSSUM: A Protocol for Surveying Social-media Users with Multimodal LLMs](http://arxiv.org/abs/2503.05529v1), 2025-03-07

(14) [Cognitive Bias Detection Using Advanced Prompt Engineering](http://arxiv.org/abs/2503.05516v1), 2025-03-07

(15) [Grammar-Based Code Representation: Is It a Worthy Pursuit for LLMs?](http://arxiv.org/abs/2503.05507v1), 2025-03-07

(16) [Statistical Guarantees of Correctness Coverage for Medical Multiple-Choice Question Answering](http://arxiv.org/abs/2503.05505v1), 2025-03-07

(17) [Benchmarking LLMs in Recommendation Tasks: A Comparative Evaluation with Conventional Recommenders](http://arxiv.org/abs/2503.05493v1), 2025-03-07

(18) [Maximum Hallucination Standards for Domain-Specific Large Language Models](http://arxiv.org/abs/2503.05481v1), 2025-03-07

(19) [The Society of HiveMind: Multi-Agent Optimization of Foundation Model Swarms to Unlock the Potential of Collective Intelligence](http://arxiv.org/abs/2503.05473v1), 2025-03-07

(20) [LLM-based Iterative Approach to Metamodeling in Automotive](http://arxiv.org/abs/2503.05449v1), 2025-03-07

(21) [Are Your LLM-based Text-to-SQL Models Secure? Exploring SQL Injection via Backdoor Attacks](http://arxiv.org/abs/2503.05445v1), 2025-03-07

(22) [An Empirical Study of Conformal Prediction in LLM with ASP Scaffolds for Robust Reasoning](http://arxiv.org/abs/2503.05439v1), 2025-03-07

(23) [Static Program Analysis Guided LLM Based Unit Test Generation](http://arxiv.org/abs/2503.05394v1), 2025-03-07

(24) [Ontology Generation using Large Language Models](http://arxiv.org/abs/2503.05388v1), 2025-03-07

(25) [R1-Omni: Explainable Omni-Multimodal Emotion Recognition with Reinforcing Learning](http://arxiv.org/abs/2503.05379v1), 2025-03-07

(26) [Shifting Perspectives: Steering Vector Ensembles for Robust Bias Mitigation in LLMs](http://arxiv.org/abs/2503.05371v1), 2025-03-07

(27) [Chain of Strategy Optimization Makes Large Language Models Better Emotional Supporter](http://arxiv.org/abs/2503.05362v1), 2025-03-07

(28) [GEMA-Score: Granular Explainable Multi-Agent Score for Radiology Report Evaluation](http://arxiv.org/abs/2503.05347v1), 2025-03-07

(29) [AutoIOT: LLM-Driven Automated Natural Language Programming for AIoT Applications](http://arxiv.org/abs/2503.05346v1), 2025-03-07

(30) [Dynamic Knowledge Integration for Evidence-Driven Counter-Argument Generation with Large Language Models](http://arxiv.org/abs/2503.05328v1), 2025-03-07

(31) [Routing for Large ML Models](http://arxiv.org/abs/2503.05324v1), 2025-03-07

(32) [Similarity-Based Domain Adaptation with LLMs](http://arxiv.org/abs/2503.05281v1), 2025-03-07

(33) [Revealing Hidden Mechanisms of Cross-Country Content Moderation with Natural Language Processing](http://arxiv.org/abs/2503.05280v1), 2025-03-07

(34) [ZOGRASCOPE: A New Benchmark for Property Graphs](http://arxiv.org/abs/2503.05268v1), 2025-03-07

(35) [Optimizing LLM Inference Throughput via Memory-aware and SLA-constrained Dynamic Batching](http://arxiv.org/abs/2503.05248v1), 2025-03-07

(36) [WritingBench: A Comprehensive Benchmark for Generative Writing](http://arxiv.org/abs/2503.05244v1), 2025-03-07

(37) [MM-StoryAgent: Immersive Narrated Storybook Video Generation with a Multi-Agent Paradigm across Text, Image and Audio](http://arxiv.org/abs/2503.05242v1), 2025-03-07

(38) [Unveiling Biases in AI: ChatGPT's Political Economy Perspectives and Human Comparisons](http://arxiv.org/abs/2503.05234v1), 2025-03-07

(39) [ARbiter: Generating Dialogue Options and Communication Support in Augmented Reality](http://arxiv.org/abs/2503.05220v1), 2025-03-07

(40) [Personalized Text Generation with Contrastive Activation Steering](http://arxiv.org/abs/2503.05213v1), 2025-03-07

(41) [Knowledge Updating? No More Model Editing! Just Selective Contextual Reasoning](http://arxiv.org/abs/2503.05212v1), 2025-03-07

(42) [Path Pooling: Train-Free Structure Enhancement for Efficient Knowledge Graph Retrieval-Augmented Generation](http://arxiv.org/abs/2503.05203v1), 2025-03-07

(43) [ORANSight-2.0: Foundational LLMs for O-RAN](http://arxiv.org/abs/2503.05200v1), 2025-03-07

(44) [Memory-augmented Query Reconstruction for LLM-based Knowledge Graph Reasoning](http://arxiv.org/abs/2503.05193v1), 2025-03-07

(45) [Rewarding Curse: Analyze and Mitigate Reward Modeling Issues for LLM Reasoning](http://arxiv.org/abs/2503.05188v1), 2025-03-07

(46) [FinTMMBench: Benchmarking Temporal-Aware Multi-Modal RAG in Finance](http://arxiv.org/abs/2503.05185v1), 2025-03-07

(47) [Sketch-of-Thought: Efficient LLM Reasoning with Adaptive Cognitive-Inspired Sketching](http://arxiv.org/abs/2503.05179v1), 2025-03-07

(48) [A Comprehensive LLM-powered Framework for Driving Intelligence Evaluation](http://arxiv.org/abs/2503.05164v1), 2025-03-07

(49) [Ensemble Debiasing Across Class and Sample Levels for Fairer Prompting Accuracy](http://arxiv.org/abs/2503.05157v1), 2025-03-07

(50) [RocketEval: Efficient Automated LLM Evaluation via Grading Checklist](http://arxiv.org/abs/2503.05142v1), 2025-03-07

(51) [Every FLOP Counts: Scaling a 300B Mixture-of-Experts LING LLM without Premium GPUs](http://arxiv.org/abs/2503.05139v1), 2025-03-07

(52) [R1-Zero's "Aha Moment" in Visual Reasoning on a 2B Non-SFT Model](http://arxiv.org/abs/2503.05132v1), 2025-03-07

(53) [Dilu: Enabling GPU Resourcing-on-Demand for Serverless DL Serving via Introspective Elasticity](http://arxiv.org/abs/2503.05130v1), 2025-03-07

(54) [Can Large Language Models Grasp Concepts in Visual Content? A Case Study on YouTube Shorts about Depression](http://dx.doi.org/10.1145/3706599.3719821), 2025-03-07

(55) [AutoTestForge: A Multidimensional Automated Testing Framework for Natural Language Processing Models](http://arxiv.org/abs/2503.05102v1), 2025-03-07

(56) [SpecServe: Efficient and SLO-Aware Large Language Model Serving with Adaptive Speculative Decoding](http://arxiv.org/abs/2503.05096v1), 2025-03-07

(57) [S2S-Arena, Evaluating Speech2Speech Protocols on Instruction Following with Paralinguistic Information](http://arxiv.org/abs/2503.05085v1), 2025-03-07

(58) [On a Connection Between Imitation Learning and RLHF](http://arxiv.org/abs/2503.05079v1), 2025-03-07

(59) [PromptPex: Automatic Test Generation for Language Model Prompts](http://arxiv.org/abs/2503.05070v1), 2025-03-07

(60) [Capacity-Aware Inference: Mitigating the Straggler Effect in Mixture of Experts](http://arxiv.org/abs/2503.05066v1), 2025-03-07

(61) [No Free Labels: Limitations of LLM-as-a-Judge Without Human Grounding](http://arxiv.org/abs/2503.05061v1), 2025-03-07

(62) [A Unified Framework with Novel Metrics for Evaluating the Effectiveness of XAI Techniques in LLMs](http://arxiv.org/abs/2503.05050v1), 2025-03-06

(63) [Dynamic-KGQA: A Scalable Framework for Generating Adaptive Question Answering Datasets](http://arxiv.org/abs/2503.05049v1), 2025-03-06

(64) [Biases in Large Language Model-Elicited Text: A Case Study in Natural Language Inference](http://arxiv.org/abs/2503.05047v1), 2025-03-06

(65) [Bridging the AI Adoption Gap: Designing an Interactive Pedagogical Agent for Higher Education Instructors](http://arxiv.org/abs/2503.05039v1), 2025-03-06

(66) [Collapse of Dense Retrievers: Short, Early, and Literal Biases Outranking Factual Evidence](http://arxiv.org/abs/2503.05037v1), 2025-03-06

(67) [Continual Pre-training of MoEs: How robust is your router?](http://arxiv.org/abs/2503.05029v1), 2025-03-06

(68) [Safety is Not Only About Refusal: Reasoning-Enhanced Fine-tuning for Interpretable LLM Safety](http://arxiv.org/abs/2503.05021v1), 2025-03-06

(69) [LLMs' Reshaping of People, Processes, Products, and Society in Software Development: A Comprehensive Exploration with Early Adopters](http://arxiv.org/abs/2503.05012v1), 2025-03-06

(70) [Leveraging Domain Knowledge at Inference Time for LLM Translation: Retrieval versus Generation](http://arxiv.org/abs/2503.05010v1), 2025-03-06

(71) [Balcony: A Lightweight Approach to Dynamic Inference of Generative Language Models](http://arxiv.org/abs/2503.05005v1), 2025-03-06

(72) [Wanda++: Pruning Large Language Models via Regional Gradients](http://arxiv.org/abs/2503.04992v1), 2025-03-06

(73) [DP-GTR: Differentially Private Prompt Protection via Group Text Rewriting](http://arxiv.org/abs/2503.04990v1), 2025-03-06

(74) [Leveraging Large Language Models For Scalable Vector Graphics Processing: A Review](http://arxiv.org/abs/2503.04983v1), 2025-03-06

(75) [LVLM-Compress-Bench: Benchmarking the Broader Impact of Large Vision-Language Model Compression](http://arxiv.org/abs/2503.04982v1), 2025-03-06

(76) [Quantifying the Relevance of Youth Research Cited in the US Policy Documents](http://dx.doi.org/10.1109/BigData62323.2024.10825004), 2025-03-06

(77) [Beyond RAG: Task-Aware KV Cache Compression for Comprehensive Knowledge Reasoning](http://arxiv.org/abs/2503.04973v1), 2025-03-06

(78) [DB-Explore: Automated Database Exploration and Instruction Synthesis for Text-to-SQL](http://arxiv.org/abs/2503.04959v1), 2025-03-06

(79) [SafeArena: Evaluating the Safety of Autonomous Web Agents](http://arxiv.org/abs/2503.04957v1), 2025-03-06

(80) [HILGEN: Hierarchically-Informed Data Generation for Biomedical NER Using Knowledgebases and Large Language Models](http://arxiv.org/abs/2503.04930v1), 2025-03-06

(81) [FirePlace: Geometric Refinements of LLM Common Sense Reasoning for 3D Object Placement](http://arxiv.org/abs/2503.04919v1), 2025-03-06

(82) [Maximizing Signal in Human-Model Preference Alignment](http://arxiv.org/abs/2503.04910v1), 2025-03-06

(83) [L$^2$M: Mutual Information Scaling Law for Long-Context Language Modeling](http://arxiv.org/abs/2503.04725v1), 2025-03-06

(84) [LLMVoX: Autoregressive Streaming Text-to-Speech Model for Any LLM](http://arxiv.org/abs/2503.04724v1), 2025-03-06

(85) [Shifting Long-Context LLMs Research from Input to Output](http://arxiv.org/abs/2503.04723v2), 2025-03-06

(86) [Enough Coin Flips Can Make LLMs Act Bayesian](http://arxiv.org/abs/2503.04722v1), 2025-03-06

(87) [Predictable Scale: Part I -- Optimal Hyperparameter Scaling Law in Large Language Model Pretraining](http://arxiv.org/abs/2503.04715v1), 2025-03-06

(88) [Universality of Layer-Level Entropy-Weighted Quantization Beyond Model Architecture and Size](http://arxiv.org/abs/2503.04704v2), 2025-03-06

(89) [UIPE: Enhancing LLM Unlearning by Removing Knowledge Related to Forgetting Targets](http://arxiv.org/abs/2503.04693v1), 2025-03-06

(90) [Quantifying the Reasoning Abilities of LLMs on Real-world Clinical Cases](http://arxiv.org/abs/2503.04691v1), 2025-03-06

(91) [DIMSUM: Discourse in Mathematical Reasoning as a Supervision Module](http://arxiv.org/abs/2503.04685v2), 2025-03-06

(92) [LLM-guided Plan and Retrieval: A Strategic Alignment for Interpretable User Satisfaction Estimation in Dialogue](http://arxiv.org/abs/2503.04675v1), 2025-03-06

(93) [Implicit Cross-Lingual Rewarding for Efficient Multilingual Preference Alignment](http://arxiv.org/abs/2503.04647v1), 2025-03-06

(94) [IFIR: A Comprehensive Benchmark for Evaluating Instruction-Following in Expert-Domain Information Retrieval](http://arxiv.org/abs/2503.04644v1), 2025-03-06

(95) [Mark Your LLM: Detecting the Misuse of Open-Source Large Language Models via Watermarking](http://arxiv.org/abs/2503.04636v1), 2025-03-06

(96) [SurveyForge: On the Outline Heuristics, Memory-Driven Generation, and Multi-dimensional Evaluation for Automated Survey Writing](http://arxiv.org/abs/2503.04629v1), 2025-03-06

(97) [START: Self-taught Reasoner with Tools](http://arxiv.org/abs/2503.04625v2), 2025-03-06

(98) [SynGraph: A Dynamic Graph-LLM Synthesis Framework for Sparse Streaming User Sentiment Modeling](http://arxiv.org/abs/2503.04619v1), 2025-03-06

(99) [Better Process Supervision with Bi-directional Rewarding Signals](http://arxiv.org/abs/2503.04618v1), 2025-03-06

(100) [HalluCounter: Reference-free LLM Hallucination Detection in the Wild!](http://arxiv.org/abs/2503.04615v1), 2025-03-06
